packages/datatrak-web-server/src/utils/getFilteredUsers.ts

```diff
@@ -21,47 +21,27 @@ const USERS_EXCLUDED_FROM_LIST = [
 
 const DEFAULT_PAGE_SIZE = 100;
 
+const usersFilter = {
+  email: { comparator: 'not in', comparisonValue: USERS_EXCLUDED_FROM_LIST },
+  [QUERY_CONJUNCTIONS.RAW]: {
+    // exclude E2E users and any internal users
+    sql: `(email NOT LIKE '%tupaia.org' AND email NOT LIKE '%beyondessential.com.au' AND email NOT LIKE '%@bes.au')`,
+  },
+} as Record<string, any>;
```

Comment:
Could `unknown` work here?

```suggestion
} as Record<string, unknown>;
```